assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -1,4 +1,9 @@
-variable "name" {
```

Comment:
If we don't care about Terraform API, we could leave it like it is and combine the name one level higher.

Reply:
We use `cluster_name` to tag the ELB target groups we create inside the `workers` module. Doing what you're suggesting would make us lose this information.